src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -500,4 +500,11 @@ object messages {
            |which cannot start with ${Red(illegalToken)}.""".stripMargin
     }
   }
+
+  case class ProperDefinitionNotFound()(implicit ctx: Context) extends Message(18) {
+    val kind = "Definition Not Found"
+    val msg = hl"""|A proper definition was not found in ${"@usecase"}"""
+    val explanation = ""
+  }
+  
```

Comment:
Great explanation, thanks for your time to do that, I will be working on it and should push a fix later :)